config_ui.go

```diff
@@ -89,11 +78,6 @@ func (c *UIConfig) ValidateAndDefault() error {
 	if c.CsrfKey != "" && len(c.CsrfKey) != 32 {
 		catcher.Add(errors.New("CSRF key must be 32 characters long"))
 	}
-	if _, err := url.Parse(c.CORSOrigin); err != nil {
```

Comment:
Why remove the validation?

Reply:
It felt unnecessary but I can add it back in if you disagree

Reply:
I'm pro-validation.

Reply:
added back in